src/util/colorScale.js

```diff
@@ -120,10 +120,20 @@ export const calcColorScale = (colorBy, controls, tree, treeToo, metadata) => {
     console.error("calcColorScale called before tree is ready.");
     error = true;
   } else if (genotype) { /* G E N O T Y P E */
-    legendValues = orderOfGenotypeAppearance(tree.nodes);
+    legendValues = orderOfGenotypeAppearance(tree.nodes, controls.mutType);
+    const trueValues = controls.mutType === "nuc" ? legendValues.filter((x) => x !== "X" && x !== "-" && x !== "N") :
+      legendValues.filter((x) => x !== "X" && x !== "-");
+    const unkValues = controls.mutType === "nuc" ? legendValues.filter((x) => x === "X" || x === "-" || x === "N") :
+      legendValues.filter((x) => x === "X" || x === "-");
+    let domain = [undefined, ...trueValues];
```

Comment:
I can't remember why we have this `undefined` leading entry, but it seems unnecessary (at least for my datasets) -- could you try your datasets with the following & see if it still works as expected?
```
let domain = [...trueValues];
let range = [...genotypeColors.slice(0, trueValues.length)];
```

Reply:
So for my couple of test sets this doesn't seem to make a difference, but I don't know if there are situations where this is possible? It seems unlikely....
I tried to test with some of the test builds that are div- and time-tree only etc, but can't seem to open them in Auspice right now. I will leave it in until I can test but if someone else is more confident that this won't hurt anything feel free to remove.